shiksha-website/shiksha-frontend/src/app/view/user/question-bank/question-bank-generation/question-bank-blue-print/question-bank-blue-print.component.html

```
@@ -20,23 +20,30 @@ <h2 class="text-xl font-bold text-content">Step 3/3: Preview & Blue Print</h2>
             </span>
           </div>
 
           <div class="space-y-3">
             <div *ngFor="let q of group.questions; let j = index" class="bg-white p-3 rounded border flex gap-3">
               <span class="text-gray-400 font-bold text-sm">{{ j + 1 }}.</span>
               <div class="flex-1">
-                <p class="text-sm text-content mb-2" [innerHTML]="q.text"></p>
+                <p class="text-sm text-content mb-2">
+                  <ng-container *ngFor="let item of questionContentItems(q)">
+                    <span *ngIf="item.contentType === 'text/plain'">{{ item.content }}</span>
+                    <img *ngIf="item.contentType !== 'text/plain'" [src]="mediaSrc(item)" class="mt-2 max-h-40 max-w-full" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)